modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFClaimForm/components/MDFClaimPage/schema/fieldValidation/getImagesValidation.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {array, object} from 'yup';
+
+import {validateDocument} from '../constants/validateDocument';
+
+export const getImagesValidation = {
+	images: array().of(
+		object()
+			.test(
+				'fileSize',
+				validateDocument.fileSize.message,
+				(imageFile) => {
+					if (imageFile && !imageFile.id) {
+						return imageFile
+							? Math.ceil(imageFile.size / 1000) <=
+									validateDocument.fileSize.maxSize
+							: false;
+					}
+
+					return true;
```

Comment:
Improve this logic, the ternary it's not necessary since the same condition is validate in if statement